fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformService.java

```diff
@@ -60,4 +61,7 @@ public interface ClientReadPlatformService {
     LocalDate retrieveClientTransferProposalDate(Long clientId);
 
     void validateClient(Long clientId);
+
```

Comment:
We dont need this

Reply:
This is a legacy code, but anyway, removed